src/common/Timer.h

```diff
@@ -90,10 +90,7 @@ class OSMTTimeVal {
         out.systime = systime - subst.systime;
         return out;
     }
-    void operator= (const OSMTTimeVal& from) {
-        usrtime = from.usrtime;
-        systime = from.systime;
-    }
```

Comment:
This is doing the same thing as default assignment operator, so it can be implicit.